pureStorageDriver/README.md

```diff
@@ -1,48 +1,12 @@
-
-## Platform and Software Dependencies
-- #### Operating Systems Supported*:
-  - CentOS 7
-  - CoreOS (Ladybug 1298.6.0 and above)
-  - RHEL 7
-  - Ubuntu 16.04
-  - Ubuntu 18.04
-- #### Environments Supported*:
-  - Kubernetes 1.13+
-    - NOTE: for Kubernetes 1.17, there is an [issue](https://github.com/kubernetes/kubernetes/issues/87852) using vxlan with Flannel or Calico 
-  - Minimum Helm version required is 3.1.0.
-  - Google Anthos 1.2.x, 1.3.x support the [stateless PSO CSI plugin](https://github.com/purestorage/helm-charts/tree/master/pure-csi) only
-  - Docker Kuberenetes Service (DKS) - based on Docker EE 3.0 with Kubernetes 1.14.3
-  - Platform9 Managed Kubernetes (PMK) - Privileged mode only
-- #### Other software dependencies:
-  - Latest linux multipath software package for your operating system (Required)
-  - Latest Filesystem utilities/drivers (XFS by default, Required)
-  - Latest iSCSI initiator software for your operating system (Optional, required for iSCSI connectivity)
-  - Latest NFS software package for your operating system (Optional, required for NFS connectivity)
-  - Latest FC initiator software for your operating system (Optional, required for FC connectivity, *FC Supported on Bare-metal K8s installations only*)
-- #### FlashArray and FlashBlade:
-  - The FlashArray and/or FlashBlade should be connected to the compute nodes using [Pure's best practices](https://support.purestorage.com/Solutions/Linux/Reference/Linux_Recommended_Settings)
-
-_* Please see release notes for details_
-
-## Additional configuration for Kubernetes 1.13 Only
-For details see the [CSI documentation](https://kubernetes-csi.github.io/docs/csi-driver-object.html). 
-In Kubernetes 1.12 and 1.13 CSI was alpha and is disabled by default. To enable the use of a CSI driver on these versions, do the following:
-
-1. Ensure the feature gate is enabled via the following Kubernetes feature flag: ```--feature-gates=CSIDriverRegistry=true```
-2. Either ensure the CSIDriver CRD is installed cluster with the following command:
-
-```bash
-kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/master/pkg/crd/manifests/csidriver.yaml
-```
+1. Pure Service Orchestrator deploys a CockroachDB datastore replicated across the provided storage backends. **Note: All licensing issues regarding this are covered by Pure Storage**
+2. Currently, there is **no upgrade supported** from previous versions that do not deploy the datastore.
+3. You **MUST** supply a unique `clusterID` in values.yaml. This was previously called `namespace.pure`. `clusterID` must be less than or equal to 22 characters in length. `clusterID` must be unique between **all** Kubernetes clusters using your Pure devices or naming conflicts will result.
+4. `helm uninstall` will perform the initial uninstallation, but some pods will continue to clean up post-installation. They should go away after cleanup is complete.
```

Comment:
Can you also add the 5th bullet point mentioned that users **MUST** install the NTP service to make the db work?
I would expect many users will report that PSO fail to work due to this issue.

Reply:
Done